src/Files/Services/Implementation/PreferencesSettingsService.cs

```diff
@@ -41,6 +41,12 @@ public bool AreSystemItemsHidden
             get => Get(true);
             set => Set(value);
         }
+        
+        public bool ShowDotFiles
+        {
+            get => Get(false);
```

Comment:
Aargh noticed too late, default value should be true right?

Reply:
Yes :( this setting was originally "hide" instead of "show", so I had set the value to false